torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -297,6 +298,56 @@ Tensor & detach_(Tensor & self) {
   return self;
 }
 
-}  // namespace VariableType
-
-}} // namespace torch::autograd
+static auto registry = torch::RegisterOperators()
+  .op(torch::RegisterOperators::options()
+    .schema("aten::resize_(Tensor(a!) self, int[] size, *, MemoryFormat? memory_format=None) -> Tensor(a!)")
+    .impl_unboxedOnlyKernel<decltype(VariableType::resize_), &VariableType::resize_>(TensorTypeId::VariableTensorId)
```

Comment:
I think it would be helpful for future readers to have an explanation why some of these are registered to `VariableTensorId`, and why others are registered as catchAll